Gemfile

```diff
@@ -68,6 +70,8 @@ group :development do
   # gem "spring"
 end
 
+gem 'letter_opener'
```

Comment:
![image](https://github.com/Ouail-01/Blog-App/assets/93039359/2cd886d5-f402-43ec-8ce4-b65838ef1459)

**[OPTIONAL]**

- [ ] Great work on adding the letter opener 😊 but when the letter opener verifies the user email then after doing it it shows the above error 😨 Kindly fix it to make your app free from errors 👍